pybossa/cache/project_stats.py

```diff
@@ -175,33 +175,20 @@ def stats_dates(project_id, period='15 day'):
 
     params = dict(project_id=project_id, period=period)
 
-    # Get all completed tasks
+    # Get all tasks with at least 1 task run and its last finish time
```

Comment:
seems like there are other problems - the query is supposed to get all "completed" tasks within last two weeks, while the query only gets tasks that have taskrun submitted within last two weeks.

Although the old query was wrong too.

Reply:
That's why I changed the comments. I would want to make sure the changes behave the same as the existing one. We can open another discussion if the existing is correct or not :)

Reply:
@ywrsusan, by looking at [line](https://github.com/bloomberg/pybossa/blob/6dba2fa761d37194e1724658f78272aae8d06aee/pybossa/cache/project_stats.py#L222) and [line](https://github.com/bloomberg/pybossa/blob/6dba2fa761d37194e1724658f78272aae8d06aee/pybossa/cache/project_stats.py#L241), I think we are not looking for the "completed" tasks. So I think even the original implementation is correct ,just too slow. But the comment was definitely wrong.

Reply:
The output of this code is eventually assigned to dayCompletedTasks - https://github.com/bloomberg/pybossa/blob/6dba2fa761d37194e1724658f78272aae8d06aee/pybossa/cache/project_stats.py#L415